catalog/src/main/resources/application.properties

```diff
@@ -1 +1,6 @@
-
+spring.jpa.hibernate.ddl-auto=update
+spring.datasource.url=jdbc:mysql://${MYSQL_HOST:localhost}:3306/catalog
```

Comment:
this must be changed to the correct url like: jdbc:mysql://localhost:3306/catalog